oecologia-australis.csl

```diff
@@ -61,21 +61,17 @@
     <choose>
       <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
         <group delimiter=", ">
-          <names variable="container-author" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
-          </names>
-          <names variable="editor translator" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
+          <names variable="editor translator" delimiter=", " suffix=" ">
+            <name delimiter=" " suffix="," and="symbol" initialize-with=". "/>
+            <label form="short" text-case="title" prefix=" "/>
           </names>
         </group>
       </if>
     </choose>
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="all"/>
+      <name and="symbol" delimiter-precedes-last="always" et-al-min="1" et-al-use-first="0" initialize-with=". " name-as-sort-order="all"/>
```

Comment:
Exclude the {exclude et-al-use-first="0"} is not possible, the test report says:

1) oecologia-australis: name nodes must have valid et-al-min and et-al-use-first attributes
     Failure/Error:
       expect(min.to_i).to be > first.to_i,
         "expected et-al-min (#{min}) to be greater than et-al-use-first (#{first})"
     
       expected et-al-min (1) to be greater than et-al-use-first (1)

So, what is the exactly problem with this? If I do not include anything here, when a reference have more than a X number of author it is automatically included "..., last author" later the 6th author, but I need that all author are included on all types of references. Could you help me?